### ISSUE TITLE
Adding Fluid+Basic Dual me interface

### ISSUE THREAD

**Issue by f1den** [AE2-UEL/Applied-Energistics-2#468]
Add more fluid items to basic AE2 mod

**Describe the feature**
Me and many other guys wanna see Better fluid crafting in 'ae2uel'.
(Taking items, parts, tiles from 'ae2uel fluid crafting')

**Reasons why it should be considered**
You have added fluid cells, fluid interface, etc.. But it is useless if you have no fluid crafting (fluid pattern encoder, etc..)

I really want to see improvements in QoL when playing with AE2, but w\o fluid crafting i am suffering.. :<<
**Additional Context**
[AE2 Fluid Crafing](https://github.com/AE2-UEL/AE2FluidCraft-Rework/blob/1.12.2/src/main/java/com/glodblock/github/common/part/PartDualInterface.java)

**Comment by RuiXuqi** [comment]
I don't think it is necessary, because you only need to put the mod file in the mods folder to let fluid join in autocrafting. Basic mod needs basic things.
